src/hb-ot-var-common.hh

```diff
@@ -1076,6 +1079,28 @@ struct TupleVariationData
     unsigned compiled_byte_size = 4;
 
     public:
+    friend void swap (tuple_variations_t& a, tuple_variations_t& b)
+    {
+      hb_swap (a.tuple_vars, b.tuple_vars);
+      hb_swap (a.point_data_map, b.point_data_map);
+      hb_swap (a.point_set_count_map, b.point_set_count_map);
+      hb_swap (a.shared_points_bytes.arrayZ, b.shared_points_bytes.arrayZ);
+      hb_swap (a.shared_points_bytes.length, b.shared_points_bytes.length);
```

Comment:
Can you just swap `shared_points_bytes` and add the swap for that to `hb_array_t`?